spec/mongoid/config_spec.rb

```diff
@@ -197,6 +197,34 @@
     end
   end
 
+  context 'when discriminator_key option' do
+    context 'is not set in the config' do
+      it 'sets the value to _type by default' do
```

Comment:
s/sets/is/. The test is reading the value, hence the heading should talk about reading and not writing.